src/utils.ts

```diff
@@ -26,14 +26,20 @@ export function withPrevAndCurrent<T, Transformed>(
   return transformedArr;
 }
 
-export function hasProperty(object: Record<string, unknown>, key: string) {
+// eslint-disable-next-line @typescript-eslint/ban-types
+export function hasProperty(object: object, key: string) {
   return Object.prototype.hasOwnProperty.call(object, key);
 }
 
 export function isJestEnv(): boolean {
-  return !!process.env.JEST_WORKER_ID;
+  return hasProperty(global, 'process') && !!process.env.JEST_WORKER_ID;
```

Comment:
Is this change needed in this PR?

Reply:
No, removed in https://github.com/software-mansion/react-native-gesture-handler/pull/1893/commits/1f7023072ff5d196ce6c479b44f67b6097e4fae8 in favor of https://github.com/software-mansion/react-native-gesture-handler/pull/1896.